daal4py/sklearn/_utils.py

```diff
@@ -117,4 +117,6 @@ def get_dtype(X):
 
 def get_number_of_types(dataframe):
```

Comment:
This refers to the number of unique types in the pandas. You get size of all type.

Why did it become necessary to change this function?

Reply:
Tests uncovered that when pd.Series is passed, error raises due to getting set from non-iterable np.dtype: `set(dtypes)`

Reply:
Ok, thanks :)